README.md

```diff
@@ -29,6 +29,8 @@ To build RustPython locally, do the following:
     $ git clone https://github.com/RustPython/RustPython
     $ cd RustPython
       # --release is needed (at least on windows) to prevent stack overflow
+
+    $ curl https://sh.rustup.rs -sSf | sh -s -- -y
     $ cargo run --release demo.py
```

Comment:
`demo.py` should be changed to `demo_closures.py`. There is no `demo.py` in this repository root.